content/docs/2.17/authentication-providers/bound-service-account-token.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Bound service account token"
++++
+
+You can pull one or more service account tokens into the trigger by defining the `serviceAccountName` of the Kubernetes service account.
+
+```yaml
+boundServiceAccountToken:                   # Optional.
+  - parameter: connectionString             # Required - Defined by the scale trigger
+    serviceAccountName: my-service-account  # Required.
+```
+
+**Assumptions:** `namespace` is in the same resource as referenced by `scaleTargetRef.name` in the ScaledObject, unless specified otherwise.
+
+## Permissions for KEDA to request service account tokens
+
+By default, the KEDA operator does not have the necessary permissions to request service account tokens from an arbitrary service account. This is to prevent a privilege escalation where a bad actor could use KEDA to request tokens on behalf of any service account in the cluster.
```

Comment:
We should document here the way how to enable this via Helm Chart, once we have the final implemenetation there